addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,7 +10976,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot and episode thumbnail for unwatched items"
```

Comment:
Then what about leaving "Show plot for unwatched items" as it is, and introduce a new option, "Show thumbnail for unwatched episodes" - this would allow increased flexibility, hiding of the plot but not necessarily the thumbnail, as I'm pretty sure some clever dick will want to be able to do that just as soon as this is merged...